lib/rehype-npm-command.ts

```
@@ -0,0 +1,18 @@
+import { visit } from "unist-util-visit";
+import { UnistNode, UnistTree } from "@/types/unist";
+
+export function rehypeNpmCommand() {
+  return (tree: UnistTree) => {
+    visit(tree, (node) => {
+      if (node.type !== "element" || node?.tagName !== "pre") {
+        return;
+      }
+
+      if (node.children && node.children[0]?.value?.startsWith("npx")) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)